pkg/services/publicdashboards/database/database.go

```diff
@@ -37,6 +37,31 @@ func ProvideStore(sqlStore *sqlstore.SQLStore) *PublicDashboardStoreImpl {
 	}
 }
 
+// Gets list of public dashboards by orgId
+func (d *PublicDashboardStoreImpl) ListPublicDashboards(ctx context.Context, orgId int64) ([]PublicDashboardListResponse, error) {
+	resp := make([]PublicDashboardListResponse, 0)
+
+	err := d.sqlStore.WithTransactionalDbSession(ctx, func(sess *sqlstore.DBSession) error {
+		//Select dashboard_public.access_token ,dashboard.title, dashboard.uid, from dashboard_public, dashboard
+		//WHERE dashboard.uid == dashboard_public.uid AND dashboard.org_id = dashboard_public.uid
+
+		sess.Table("dashboard_public").
+			Join("LEFT", "dashboard", "dashboard.uid = dashboard_public.dashboard_uid").
+			Cols("dashboard_public.uid", "dashboard_public.access_token", "dashboard_public.dashboard_uid", "dashboard_public.is_enabled", "dashboard.title").
+			Where("dashboard_public.org_id = dashboard.org_id").
```

Comment:
Don't think this is needed since were already doing an explicit join on dashboard uid